tasks/run-matlab-command/v0/main.ts

```diff
@@ -11,13 +11,19 @@ async function run() {
     try {
         taskLib.setResourcePath(path.join( __dirname, "task.json"));
         const command = taskLib.getInput("command", true);
-        await runCommand(command as string);
+        const startupopts = taskLib.getInput("startup-options");
+
+        if (startupopts) {
+            await runCommand(command as string, startupopts.split(" "));
+        } else {
+            await runCommand(command as string);
+        }
```

Comment:
Might need a small tweak because this is untested code but I think this can be made a little more typescript-y:
```suggestion
        const startupOpts: string = taskLib.getInput("startupOptions") || "";

        await runCommand(command as string, startupOpts.split(" "));
```
I think we can add a type annotation on the declairation of command above too, to get rid of the "as string"

Reply:
Hmm... maybe this isn't quite right. I think then we're always passing the args param to runCommand. I feel like there's a pretty typescript-y solution here...

Reply:
Yeah that looks much cleaner!